lib/matplotlib/backends/backend_qt5.py

```diff
@@ -486,7 +486,7 @@ def drawRectangle(self, rect):
         # Draw the zoom rectangle to the QPainter.  _draw_rect_callback needs
         # to be called at the end of paintEvent.
         if rect is not None:
-            x0, y0, w, h = [pt / self._dpi_ratio for pt in rect]
+            x0, y0, w, h = map(int, [pt / self._dpi_ratio for pt in rect])
```

Comment:
You could just use `int` directly?
```suggestion
            x0, y0, w, h = [int(pt / self._dpi_ratio) for pt in rect]
```

Reply:
fair, `map` is what popped into my head first..